Readability.js

```diff
@@ -904,6 +904,12 @@ Readability.prototype = {
           continue;
         }
 
+        // User is not able to see elements applied with both "aria-modla = true" and "role = dialog"
+        if (node.getAttribute("aria-modal") == "true" && node.getAttribute("role") == "dialog") {
+          node = this._removeAndGetNext(node);
+          continue;
+        }
```

Comment:
This should be added to the check in [isProbablyVisible](https://github.com/tthhtao/readability/blob/2edee0ee6224b9c0ed94f81832cb5e2678d0f0d4/Readability.js#L2206).

Reply:
We should remove this check since it should be handled in the one above now